Library/Formula/libimobiledevice.rb

```diff
@@ -26,14 +26,20 @@ class Libimobiledevice < Formula
   depends_on "usbmuxd"
   depends_on "openssl"
 
+  option "with-cython", "Compile with Cython support - for Python bindings"
```

Comment:
Looks from `libplist.rb` that this should be `with-python` and have a `depends_on :python => :optional`